### ISSUE TITLE
Unable to resolve module AssetRegistry

### ISSUE THREAD

**Issue by ajoshdee** [aksonov/react-native-router-flux#416]
Hi, I'm getting this error: 
Unable to resolve module AssetRegistry from /Users/.../React-native.../.../node_modules/react-native-router-flux/src/back_chevron.png: Unable to find this module in its module map or any of the node_modules directories under /Users/node_modules/AssetRegistry and its parent directories

**Comment by levinqdl** [comment]
@ajoshdee I have the same issue, too. How do you fix the problem?

**Comment by aksonov** [comment]
@levinqdl Are you using 0.23 ?

**Comment by levinqdl** [comment]
@aksonov yes

**Comment by aksonov** [comment]
0.22 is supported now. 0.23 is not even released, i can't  (and should not)  make sure that my component will work with release candidates...

**Comment by levinqdl** [comment]
Yeah, I understand. But 0.22 have a really bad nav transition performance, so I want to try 0.23. Anyway, I can wait...

**Comment by caledhwa** [comment]
I confirm that this doesn't work with 0.23.0-rc4, but I also agree with @aksonov that supporting RC's seems like too far ahead. Also, @levinqdl has somewhat of a point too. Basically, the navigator stuff seems broken for ANDROID in 0.22. I confirmed that 0.23 did fix the performance issues with navigator and works well with react-native-router-flux. However, once you upgrade to the RC, a bunch of other things break including not being able to run at all in iOS. So, hopefully, all of this gets resolved with a 0.23 final release. We shall see.

**Comment by felipemartim** [comment]
Setting hideNavBar={true} on the root scene fixed it for me.

**Comment by viper1029** [comment]
If you need the navBar, providing your own back button also works.
```
<Router>
        <Scene key="root" renderBackButton={this.renderBackButton}>
          <Scene key="AboutUs" component={AboutUs} title="AboutUs"/>
        </Scene>
</Router>
```

**Comment by evollu** [comment]
looks like they are going to fix in 0.24.0-rc2
https://github.com/facebook/react-native/issues/6800
